sw/device/lib/dif/autogen/dif_lc_ctrl_autogen.inc

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is auto-generated.
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_AUTOGEN_DIF_LC_CTRL_AUTOGEN_INC_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_AUTOGEN_DIF_LC_CTRL_AUTOGEN_INC_
+
+/**
+ * @file
+ * @brief <a href="/hw/ip/lc_ctrl/doc/">LC_CTRL</a> Device Interface Functions
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_base.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * A handle to lc_ctrl.
+ *
+ * This type should be treated as opaque by users.
+ */
+typedef struct dif_lc_ctrl {
+  /**
+   * The base address for the LC_CTRL hardware registers.
```

Comment:
Hmm, something seems wrong here. lc_ctrl should be "Lifecycle Controller", right?

Reply:
This is for the same reason stated here: https://github.com/lowRISC/opentitan/pull/8280#discussion_r713168093, though admittedly, it could be lower snake case instead of upper.

Fixed.